### ISSUE TITLE
add transforms to DataLoader

### ISSUE THREAD

**Issue by pajotarthur** [pytorch/pytorch#437]
Could it be a good idea to add 'transforms' argument directly into torch.utils.data.DataLoader . 
Something like : 
`torch.utils.data.DataLoader(dataset, batch_size=opt.batchSize, transforms=transforms.Normalize(),
                                        shuffle=True, num_workers=int(opt.workers))`

It could be useful for inverting axis and feed the data to RNN.

**Comment by soumith** [comment]
transforms need to be written via a custom collate function if needed on the DataLoader side.
